.circleci/config.yml

```diff
@@ -154,6 +190,55 @@ jobs:
               Write-Host "Forcing build failure due to test failure(s) :'("
               $host.SetShouldExit(1)
             }
+  publish:
+    <<: *defaults
+    steps:
+      - add_ssh_keys: *ssh-config
+      - checkout
+      - restore_cache: *restore_cache
+      - run: *gh-config
+      - run: *install
+      - run:
+          name: 'Install build dependencies'
+          command: |
+            npm install -g lerna@3.13.1
+            npm install -g vsce
+            npm install -g shelljs@0.8.3
+      - run: *gus-create-change-mgmt
+      - run:
+          name: 'Compile in production mode'
+          environment:
+            - NODE_ENV: production
+          command: |
+            npm run compile
+      - run:
+          name: 'Packaging extensions'
+          command: npm run vscode:package
+      - run:
+          name: 'Generating the SHA256 and appending to the file.'
+          command: npm run vscode:sha256
+      - run:
+          name: 'Concatenating the contents to the proper SHA256.md'
+          command: scripts/concatenate-sha256.js
+      - run:
+          name: 'Removing the temp SHA256 file.'
+          command: rm ./SHA256
+      - run:
+          name: 'Adding the SHA256 to git.'
+          command: |
+            git add SHA256.md
+            git commit -m "[skip ci] Updated SHA256"
```

Comment:
Adding `[skip ci]` to the commit message allows us to skip building/releasing again after pushing these updates back to the master branch.